<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopupChatWindow</class>
 <widget class="QMainWindow" name="PopupChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>451</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <widget class="QWidget" name="chatcentralwidget">
   <layout class="QHBoxLayout" name="chatcentralLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="ChatTabWidget" name="tabWidget">
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="chatstatusbar"/>
  <widget class="QToolBar" name="chattoolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAvatar"/>
   <addaction name="actionColor"/>
   <addaction name="actionSetOnTop"/>
   <addaction name="actionDockTab"/>
   <addaction name="actionUndockTab"/>
  </widget>
  <action name="actionAvatar">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/no_avatar_70.png</normaloff>:/images/no_avatar_70.png</iconset>
   </property>
   <property name="text">
    <string>Avatar</string>
   </property>
   <property name="toolTip">
    <string>Set your Avatar Picture</string>
   </property>
  </action>
  <action name="actionDockTab">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/tab-dock.png</normaloff>:/images/tab-dock.png</iconset>
   </property>
   <property name="text">
    <string>Dock tab</string>
   </property>
   <property name="toolTip">
    <string>Dock tab</string>
   </property>
  </action>
  <action name="actionUndockTab">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/tab-undock.png</normaloff>:/images/tab-undock.png</iconset>
   </property>
   <property name="text">
    <string>Undock tab</string>
   </property>
   <property name="toolTip">
    <string>Undock tab</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/highlight.png</normaloff>:/images/highlight.png</iconset>
   </property>
   <property name="text">
    <string>Set Chat Window Color</string>
   </property>
   <property name="toolTip">
    <string>Set Chat Window Color</string>
   </property>
  </action>
  <action name="actionSetOnTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/pin32.png</normaloff>:/images/pin32.png</iconset>
   </property>
   <property name="text">
    <string>Set window on top</string>
   </property>
   <property name="toolTip">
    <string>Set window on top</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChatTabWidget</class>
   <extends>QTabWidget</extends>
   <header location="global">gui/chat/ChatTabWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
